views/api/index.pug

```diff
@@ -18,7 +18,7 @@ block content
             img(src='https://i.imgur.com/jiztYCH.png', height=40, style='padding: 0px 10px 0px 0px')
             | Facebook
     .col-md-4
-      a(href='/api/foursquare', style='color: #fff')
+      a(href='/api/foursquare', style='color: #000000')
```

Comment:
I think the rest of the style colors are using 3 chars.  Let's keep the code consistent here as well.

Reply:
okay, will change it.